### DISCUSSION TITLE
Setting grammarly check to manual only

### DISCUSSION THREAD

**Discussion by thorsent** [znck/grammarly#339] [Q&A]
Grammarly on .ts,.js etc (or even technical documents in markdown) results in a lot of red-line. This obscures errors from type-checking, es-lint, and other plugins. Ideally, the grammarly checking would be initially paused, with a button to toggle it on (when you're ready to review grammar).

Currently, the opposite appears to be the case. There's a button in the icon in the bottom-right corner of the screen, but checking is turned on initially and then I have to toggle it _off_. Is there a way to reverse this behavior?

I thought that the "Start test checking session in paused state" setting would accomplish this, but I have it turned off and the extension is still running whenever I load a new file.

**Comment by tomasrei** [comment]
+1 

I use vscode for latex compilation, and mixing Grammarly with compilation errors is confusing. It would indeed be nice to set the default to off for this extension.

Another option, that exist for other spell-check extensions, is to change the messages from error to warning (i.e. from red circle to yellow triangle). 

Also, thanks for a great extension!
